src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -135,7 +136,20 @@ const QuickMarcEditor = ({
     handleSubmit(e).then((updatedRecord) => {
       handleSubmitResponse(updatedRecord);
     });
-  }, [deletedRecords, handleSubmit, handleSubmitResponse]);
+  }, [deletedRecords.length, handleSubmit, handleSubmitResponse]);
+
+  const handleFooterSave = useCallback((e, isKeepEditing = false) => {
+    continueAfterSave.current = isKeepEditing;
+
+    if (marcType === MARC_TYPES.AUTHORITY && numOfLinks > 0 && are010Or1xxUpdated(initialValues.records, records)) {
+      setIsUpdate0101xxfieldsAuthRecModalOpen(true);
+
+      // eslint-disable-next-line no-useless-return
+      return;
+    } else {
+      confirmSubmit(e);
+    }
```

Comment:
You do not need `return` here if you use `else`
```suggestion
    if (marcType === MARC_TYPES.AUTHORITY && numOfLinks > 0 && are010Or1xxUpdated(initialValues.records, records)) {
      setIsUpdate0101xxfieldsAuthRecModalOpen(true);
    } else {
      confirmSubmit(e);
    }
```